crates/fluvio-version/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+/// The `git rev-parse HEAD` of the current build of Fluvio.
+pub const GIT_HASH: &str = env!("GIT_HASH");
+
+/// The current version of Fluvio.
+pub const VERSION: &str = include_str!("../../../VERSION");
+
+pub mod built_info {
```

Comment:
Perhaps nit, but I would use `build` over `built_info` given that it is more conventional?

For instance, in version context, you tend to use `build` for the build number/identifier.

Reply:
Agreed the module name is a little obtuse, the name is lifted directly from `fluvio-socket/src/lib.rs`, and comes from the fact that it is the info output by the `built` library. But I will change it.